src/Jenssegers/Mongodb/Schema/Blueprint.php

```diff
@@ -235,6 +238,41 @@ public function sparse_and_unique($columns = null, $options = [])
         return $this;
     }
 
+    /**
+     * Check whether the given index exists.
+     *
+     * @param  string|array  $indexOrColumns
+     * @return bool
+     */
+    public function hasIndex($indexOrColumns = null)
+    {
+        $indexOrColumns = $this->transformColumns($indexOrColumns);
+        foreach ($this->collection->listIndexes() as $index) {
+            if ($index->getName() == $indexOrColumns) {
```

Comment:
What will happened if `$indexOrColumns` will be array?

Reply:
I will update it. I guess that code change was made against an older version of laravel-mongodb.